test/format/t.c

```diff
@@ -237,18 +240,6 @@ main(int argc, char *argv[])
     if (quiet_flag || !isatty(1))
         g.c_quiet = 1;
 
-    /*
-     * Multithreaded runs can be replayed: it's useful and we'll get the configuration correct.
-     * Obviously the order of operations changes, warn the user.
```

Comment:
I'm not sure how the removal of the replay functionality fits into adding format logging to WT log files. Is this a gratuitous change?

Reply:
Yes and no. I've been thinking about removing `replay` for awhile, but when I took a run at making it work in the new format structure, it was hard to get right. Given I don't think it's worth keeping, I didn't see any reason to do that work.